### ISSUE TITLE
Great Job. Some Questions.

### ISSUE THREAD

**Issue by glowbro** [sstativa/mfc-node#1]
Hey, great job on this script. Got it running no problem.

A couple of questions.

1. Is this recording the mobile stream? The stream capture quality looks to be very low quality.
If you are capturing the mobile stream, is it possible to add an option to capture the HD stream?

2. The model list becomes quite confusing since you need to add the models ID. Is it possible to include the models name next to the ID, so it's easy to figure out who is who when revising the models in config.yml.

Also, would it be possible, where if you add an asterisk next to a name, it won't record the model?

For example:

models:
  - 05863493945 (mfc model name)
  - 84859393944 (mfc model name)*  <--won't record if asterisk present

Thanks for the great work!

**Comment by pusspounder** [comment]
1. Yes it is recording the following:
```
'http://video' + (model.camserv - 500) + '.myfreecams.com:1935/NxServer/ngrp:mfc_' + (100000000 + model.uid) + '.f4v_mobile/playlist.m3u8',
```
My testing showed that some streams are regular old 320x240, some are 800x600. 

From here:

http://stream-recorder.com/forum/showpost.php?p=85833&postcount=611

```
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-STREAM-INF:BANDWIDTH=743749,CODECS="avc1.77.31,mp4a.40.2",**RESOLUTION=800x600**
chunklist_w2147401229_b743749.m3u8?nc=1469205041856
#EXT-X-STREAM-INF:BANDWIDTH=582000,CODECS="avc1.66.30,mp4a.40.2",RESOLUTION=320x240
chunklist_w2147401229_b532000.m3u8?nc=1469205041856
```

I do not know how to tell FFMPEG to download the stream with the highest
 resolution. So this needs more investigating by FFMPEG experts.
 
FFMPEG
https://www.ffmpeg.org/documentation.html

Also I'm not a NodeJS expert, but one can possibly use Livestreamer instead of FFMPEG. Livestreamer uses RTMPDump instead of FFMPEG, and it can choose the **best** resolution.

The syntax is (for Windows):

`livestreamer-v1.12.2-win32\livestreamer.exe "URL" best -o output.ts`

TLDR: the answer is: I don't know :(

2. Try adding comments in `config.yml`. Works here.

```
models:
  - 20249367 #test
  - 17030607 #oh dem girls!
  - 20346887
```

**Comment by sstativa** [comment]
Hi,

1) Yes, the the script uses mobile feed because it is easier to implement. RTMPDump is more complex and require `rtmpdump` on user's machine, and installing and compiling of `rtmpdump` can be a challenge for some users. I'll have a look later, probably I would manage to make rtmpdump version for HD streams.

2) The direct editing of models in `config.yml` is possible, but the best and the right way will be to use sections `includeModels` and `excludeModels` of `updates.yml` file. The script looks for changes in this file and if it finds some it moves them to `config.yml` then applies them. So, in this case, you don't need to restart the script after editing `config.yml`.

The script cannot write comments in `config.yml` file, therefore I'll not recommend to leave any comments in `config.yml` file. When the script finds changes in `updates.yml` and moves them to `config.yml` all your comments will be lost. 

However, based on your suggestions I changed the structure on `models` from "flat list" to collection of "records". The script will convert old `models` to new version automatically at the start. Each record will include:
* `uid` - id of the model (this field is required)
*  `nm` - name of the model (optional, if it's not set the script will use real model's name)
* `excluded` - if it is set and is equal true, the script will ignore this model and never start the capturing process (this is alternative to your `*` character).
